### PR TITLE
Integrated ICO by Densify recommendation the the EC2 ICO Example

### PR DESCRIPTION
**Pull request by daveshrestha-intel**
Created example for AWS EC2 Linux VM with ICO by Densify recommendation

### CODE REVIEW

File: examples/amazon-linux-ec2-default-vpc-ico-by-densify/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+module "ec2-vm" {
+  source   = "intel/aws-vm/intel"
+  key_name = aws_key_pair.TF_key.key_name
+  tags = {
+    Name     = "my-test-vm-${random_id.rid.dec}"
+    Owner    = "OwnerName-${random_id.rid.dec}",
+    Duration = "2"
+  }
+}
```
Comment (lucasmelogithub) [inline examples/amazon-linux-ec2-default-vpc-ico-by-densify/README.md:85]: We should make this look like main.tf

File: examples/amazon-linux-ec2-default-vpc-ico-by-densify/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+# Intel® Optimized Cloud Modules for Terraform
+
+© Copyright 2022, Intel Corporation
+
+## Terraform Intel AWS VM - Linux VM in Default VPC
+<p align="center">
+  <img src="https://github.com/intel/terraform-intel-aws-mysql/blob/main/images/aws-ec2-ico.png?raw=true" alt="Intel + Densify Logo" width="250"/>
+</p>
+
+This example creates creates AWS EC2 instance on Linux Operating System in the default VPC using recommended instance from Intel Cloud Optimizer by Densify. Intel® Cloud Optimizer is a collaboration between Densify and Intel targeted at getting you the most from your cloud investment. Intel Cloud Optimizer by Densify helps customers optimize their cloud investments and ensure optimal performance for every workload. Using this example requires a densify_recommndations.auto.tfvars file. You are expected to generate this file so this is a sample file only. In this sample file we will use example of recommended instance type of db.m6i.xlarge.
```
Comment (lucasmelogithub) [inline examples/amazon-linux-ec2-default-vpc-ico-by-densify/README.md:14]: Can we add more bullet points and links on how to: 
- Find out more 
- Contact Densify/Intel

**Review by daveshrestha-intel — COMMENTED** [review]
Done

**Comment by daveshrestha-intel** [comment]
Updated with ICO program offers and udpaetd main example in the readme

**Review by lucasmelogithub — APPROVED** [review]
LGTM

**Pull request merged**